files/common/usr/bin/ebs-nvme-mapping

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+#
+# TODO: ask karyn about license.
+# This script is from https://github.com/oogali/ebs-automatic-nvme-mapping,
+# which is under the MIT License
+#
```

Comment:
Once we get this clarified, we'll also want to add our copyright since we are modifying the script slightly.